modules/microshift-install-rpms.adoc

```diff
@@ -20,17 +20,15 @@ Use the following procedure to install {microshift-short} from an RPM package.
 [source,terminal,subs="attributes+"]
 ----
 $ sudo subscription-manager repos \
-    --enable rhocp-<4.18>-for-<9>-$(uname -m)-rpms \ # <1>
-    --enable fast-datapath-for-<9>-$(uname -m)-rpms # <2>
+    --enable rhocp-${ocp-version}-for-rhel-${op-system-version-major}-$(uname -m)-rpms \
+    --enable fast-datapath-for-rhel-${op-system-version-major}-$(uname -m)-rpms # <2>
 ----
-<1> Replace _<4.18>_ and _<9>_ with the compatible versions of your {microshift-short} and {op-system-base-full}.
-<2> Replace  _<9>_ with the compatible version of {op-system-base}.
 
 . For extended support (EUS) releases, also enable the EUS repositories by running the following command:
 +
 [source,terminal]
 ----
-`$ sudo subscription-manager repos \
+$ sudo subscription-manager repos \
     --enable rhel-<9>-for-x86_64-appstream-eus-rpms \ # <1>
```

Comment:
Let's remove `<>`  from `<9>`? It's very inconvenient to copy-paste and then edit this code.
Even it we do not have a variable that denotes RHEL major version, we can keep it hardcoded.

Also, the comments <1> and <2> do not seem to be necessary. We only support v9 of RHEL atm.

Reply:
We do, actually, have an attribute. We will likely need a second one when we get to RHEL 10, but can decide how to handle that then. Updated.